src/components/ZclAttributeReportingManager.vue

```diff
@@ -221,6 +221,16 @@ export default {
                   .toLowerCase()
                   .includes(this.individualClusterFilterString.toLowerCase())
           })
+
+        resp.forEach((attribute) => {
```

Comment:
I notice ```attributeData``` is used here ```    <q-table
      class="my-sticky-header-table"
      :rows="attributeData"```

Will this run whenever the user toggles and changes ```this.selectedReporting``` and change it back?

An easy fix, to explain, would be adding a flag like "initialSetup" or something along those lines. After the code here runs, you can have "initialSetup" set to false then add a condition where this code only runs once.

Reply:
I did not expect the code to be run whenever ```selectedReporting changes```, and that causes the problem. A flag seems an easy solution here!